client-api/src/main/kotlin/org/ostelco/prime/client/api/TopupModule.kt

```diff
@@ -37,27 +39,33 @@ class TopupModule : PrimeModule {
 
     private val storage by lazy { getResource<Storage>() }
     private val ocsSubscriberService by lazy { getResource<OcsSubscriberService>() }
+    private val LOG by logger()
 
     override fun init(env: Environment) {
 
         val dao = SubscriberDAOImpl(storage, ocsSubscriberService)
         val jerseyEnv = env.jersey()
 
+        val client: Client = JerseyClientBuilder(env)
+                .using(config.jerseyClientConfiguration)
+                .using(ObjectMapper()
+                        .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false))
+                .build(env.getName())
+        client.property(ClientProperties.CONNECT_TIMEOUT, 2000)
```

Comment:
Not really needed as the timeout values are pulled in from the config file through the `config.jerseyClientConfiguration` call. If values different from Dropwizards default are needed then it is just to update the config. See:
https://www.dropwizard.io/1.3.2/docs/manual/configuration.html#man-configuration-clients-http